integration/tests/lib/block_state.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/onflow/flow-go/model/messages"
 )
 
-const blockStateTimeout = 60 * time.Second
+const blockStateTimeout = 120 * time.Second
```

Comment:
why the increase in the timeout? Is it because we're giving 2 minutes for the attacker to register?

Reply:
yes, also realized Wintermute is the heaviest test with 3 ENs and 4 VNs. Hence, bumping up to 2 minutes to account for potential run time slow down.